src/pool/rpc.h

```diff
@@ -185,8 +185,8 @@ CRT_RPC_DECLARE(pool_create, DAOS_ISEQ_POOL_CREATE, DAOS_OSEQ_POOL_CREATE)
 
 #define DAOS_OSEQ_POOL_CONNECT	/* output fields */		 \
 	((struct pool_op_out)	(pco_op)		CRT_VAR) \
-	((struct daos_pool_space) (pco_space)		CRT_VAR) \
-	((struct daos_rebuild_status) (pco_rebuild_st)	CRT_VAR) \
+	((struct daos_pool_space) (pco_space)		CRT_RAW) \
```

Comment:
why is pco_space here changed to CRT_RAW? In other instances (e.g. line 212) it is still declared as CRT_VAR and we still have custom pack function for it

Reply:
Thanks to point this. I just miss this instance and one more.